pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -55,6 +53,64 @@ tile = rf.select(rf_tile("proj_raster")).first()[0]
 display(tile)
 ```
 
+## Multiple Singleband Rasters
+
+In this example, we show reading [two bands](https://en.wikipedia.org/wiki/Multispectral_image) of [Landsat 8](https://landsat.gsfc.nasa.gov/landsat-8/) imagery (red and near-infrared), combining them with `rf_normalized_difference` to compute NDVI. As described in the section on @ref:[catalogs](raster-catalogs.md), image URIs in a single row are assumed to be from the same scene/granule, and therefore compatible. This pattern is commonly used when multiple bands are stored in separate files.
+
+```python, multi_singleband
+bands = [f'B{b}' for b in [4, 5]]
+uris = [f'https://landsat-pds.s3.us-west-2.amazonaws.com/c1/L8/014/032/LC08_L1TP_014032_20190720_20190731_01_T1/LC08_L1TP_014032_20190720_20190731_01_T1_{b}.TIF' for b in bands]
+catalog = ','.join(bands) + '\n' + ','.join(uris)
+
+rf = spark.read.raster(catalog, bands) \
+    .withColumnRenamed('B4', 'red').withColumnRenamed('B5', 'NIR') \
+    .withColumn('longitude_latitude', st_reproject(st_centroid(rf_geometry('red')), rf_crs('red'), lit('EPSG:4326'))) \
```

Comment:
maybe worth breaking this out with a few more code comments explaining what we are doing with each column.